index.js

```diff
@@ -52,7 +52,7 @@ async function run() {
 			owner,
 			name: releaseTitle.replace('{tag}', pushedTag),
 			tag_name: pushedTag, // eslint-disable-line camelcase
-			body: await generateReleaseNotes({range, exclude, commitTemplate, releaseTemplate, dateFormat, reverseSort}),
+			body: await generateReleaseNotes({octokit, owner, repo, range, exclude, commitTemplate, releaseTemplate, dateFormat, reverseSort}),
```

Comment:
The object should now be on multiple lines

Reply:
Like that?
```suggestion
			body: await generateReleaseNotes({
				octokit, owner, repo, range, exclude,
				commitTemplate, releaseTemplate, dateFormat, reverseSort
			}),
```

Reply:
Fixed in 343aa58